static/js/github-configuration.js

```diff
@@ -1,4 +1,14 @@
-/* globals $ */
+/**
+ * This method passes a value to the Parent Window,
+ * which is used to change the url of the Parent Window
+ */
+const updateParentWindowURL = () => {
+  if (window.opener != null && !window.opener.closed && window.opener.document.getElementById("redirectToGitHubConfigPage")) {
+    window.opener.document.getElementById("redirectToGitHubConfigPage").value = true;
+  }
+  window.close();
+};
```

Comment:
This screams of a hack to me.  What is it you're trying to do?

Reply:
I'm trying to pass a value from child window to the parent window, which determines the url of the parent window. Couldn't think of a better way than this...

Reply:
but why? what's the use case?

Reply:
We want to change the url of the parent window once the user connects to an app through the child window.

This is our current situation:
https://user-images.githubusercontent.com/37155488/178183440-efd650c0-b059-4431-b766-e65c9439bd94.mov
 
This is what we want:
https://user-images.githubusercontent.com/13076255/178188032-c2947b80-b2d0-4535-94bf-d5144502860c.mov

Reply:
yeah, there's a better way of doing that with the `openChildWindow` function in jira-configuration.js file.  ie. when child closes, go back to previous view instead of just reload :)

Reply:
oh wait, you already have that below! you don't need this change at all or need to pass any information